packages/gatsby-plugin-manifest/src/gatsby-node.js

```diff
@@ -54,6 +54,11 @@ exports.onPostBootstrap = async (args, pluginOptions) => {
     manifest.icons = defaultIcons
   }
 
+  // Specify extra options for each icon (if requested).
+  manifest.icons.forEach(icon => {
```

Comment:
Another nit (feel free to ignore!)

I typically prefer map over forEach since it discourages mutations to Objects (which after many more additions, may manifest in hard to debug errors)

```
manifest.icons = manifest.icons.map(icon => ({
  ...icon,
  ...pluginOptions.icon_options,
}))
```

Reply:
I was thinking about the same approach, but in such a small variable scope, I don't think that mutations may cause any trouble. After all, mutations are also more performant and in this particular case, they even result in shorter code with better expression of intents.